CONTRIBUTING.zh-CN.md

```
@@ -0,0 +1,126 @@
+
+贡献流程结束，感谢你的贡献！
+
+## 加入开源社区
+
+如果你对我们的 Fluent Editor 开源项目感兴趣，欢迎通过以下方式加入我们的开源社区。
+
+- 添加官方小助手微信：opentiny-official（备注：Fluent Editor），加入我们的技术交流群
+
+如果你给 Fluent Editor 提交过 Issue 或 PR，请在 Issue 或 PR 下面评论以下内容，添加自己到贡献者列表里。
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)